package/kernel/linux/modules/usb.mk

```diff
@@ -1216,6 +1216,7 @@ $(eval $(call KernelPackage,usb-net-mcs7830))
 
 define KernelPackage/usb-net-smsc95xx
   TITLE:=SMSC LAN95XX based USB 2.0 10/100 ethernet devices
+  DEPENDS:=+LINUX_5_10:kmod-libphy
```

Comment:
This fixes kmod-usb-net-smsc95xx. You should change commit title.

This commit is similar to the one in my [PR.](https://github.com/openwrt/openwrt/pull/4002) I missed this PR due to incorrect commit title.